src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,27 +8,61 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
+    public static final String EXAMPLE = "123, some street, #00-00, 000000";
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    
+    private static final int BLOCK_INDEX = 0;
+    private static final int STREET_INDEX = 1;
+    private static final int UNIT_INDEX = 2;
+    private static final int POSTAL_CODE_INDEX = 3;
 
-    public final String value;
+    private final Block block_;
```

Comment:
Note that having a final object does not mean that their internal values cannot be changed.